test/conftest.py

```diff
@@ -0,0 +1,143 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
+
+# pylint: disable=protected-access
+
+"""pytest fixtures for KernelCI API"""
+
+from unittest.mock import AsyncMock
```

Comment:
Put imports in the alphabetical order

Reply:
The same goes for this too.

Reply:
So, we should group standard library, local imports and third party ones separately and put an empty line between each group.

Reply:
Done.